src/client/src/ui/blotter/epics/blotterServiceEpic.ts

```diff
@@ -60,4 +60,18 @@ const connectBlotterToNotifications: ApplicationEpic = (action$, state$, { platf
     ignoreElements(),
   )
 
-export const publishBlotterEpic = combineEpics(connectBlotterToExcel, connectBlotterToNotifications)
+export const requestBrowserNotificationPermission: ApplicationEpic = (action$, state$, { platform }) =>
+  action$.pipe(
+    applicationConnected,
+    delay(3000),
+    tap(() => {
+      Notification.requestPermission(permission => {
+        if (permission === 'granted') {
+          console.log('Notification permission granted')
```

Comment:
As above enum for permission value.  Also is this the required behaviour? Do we need to react to a permission not granted? Either way I'm questioning the console.log!

Reply:
This has been removed as it is not needed, it is just Notification.requestPermission() now